src/ui/views/ManualDownload.svelte

```diff
@@ -1,23 +1,102 @@
 <script>
   const { ipcRenderer } = require("electron");
+  const path = require('path');
+  const fs = require('fs');
+  
   import {
     manualDownloadFileData,
     manualDownloadGroup,
     eventObject
   } from "../../stores.mjs";
 
-  let downloadedFile;
   let checkingFile = false;
+  let selectedDirectory = "";
+  let constructedFilePath = "";
+  let errorMessage = "";
+
+  // Pick directory and construct full file path
+  async function selectDirectoryAndConstructPath() {
+    errorMessage = ""; // Clear previous errors
+    
+    try {
+      const result = await ipcRenderer.invoke('show-file-dialog', {
+        properties: ['openDirectory'],
+        defaultPath: require('os').homedir() + '/Downloads',
```

Comment:
Same issue as the other call

Reply:
In web browsers and most Electron configurations, ```File.path``` doesn't exist or is ```undefined``` for security reasons. Browsers don't expose full file system paths to prevent malicious websites from learning about your file structure. So you need to find an other way to extract the directory path. Since i cant change the UI and add a button (that would solve the problem), i am "stuck" to add a fixed direcly path. Same as the other call, which was actually a test call.